packages/site/cms-site/src/cookies/useDevCookieApi.ts

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { useLocalStorage } from "usehooks-ts";
+
+import { CookieApiHook } from "./CookieApiHook";
+
+const localStorageCookieApiKey = "comet-dev-cookie-api-consented-cookies";
+
+/**
+ * Only for use in development environment.
+ */
+export const useDevCookieApi: CookieApiHook = (): ReturnType<CookieApiHook> => {
+    const [consentedCookies, setConsentedCookies] = useLocalStorage<string[]>(localStorageCookieApiKey, []);
+
+    React.useEffect(() => {
+        const storedCookies = window.localStorage.getItem(localStorageCookieApiKey);
+        const cookiesList = JSON.parse(storedCookies ?? "[]");
+        logCookieUpdate(cookiesList);
+    }, []);
+
+    const openCookieSettings = React.useCallback(() => {
+        const cookies = prompt('Define consented cookies (separated by ","):', consentedCookies.join(",")) ?? "";
+        const cookiesList = cookies.split(",").map((cookie) => cookie.trim());
+        setConsentedCookies(cookiesList);
+        logCookieUpdate(cookiesList);
+    }, [consentedCookies, setConsentedCookies]);
+
+    React.useEffect(() => {
+        // @ts-expect-error TODO: Can this be fixed?
```

Comment:
You can defined this in the window I suppose

Reply:
What do you mean exactly? 
Do we define/change the type of window somewhere already?

Reply:
I meant extending the window object. Something like this: https://www.squash.io/tutorial-extending-window-object-in-typescript/. Or is this a different error?

Reply:
I fixed it like this now: 228edabc5aa84e1dffef4e1185ab832a41649bab